bigtable/client/table.cc

```diff
@@ -107,5 +107,9 @@ void Table::BulkApply(BulkMutation&& mut) {
   }
 }
 
+RowReader Table::ReadRows(RowSet row_set, int rows_limit, Filter filter) {
```

Comment:
You return a `RowReader` but the class is defined in `bigtable/client/internal` and presumably in the `internal` namespace?

Reply:
done